api-reference/v1.0/resources/groups_overview.md

```diff
@@ -0,0 +1,125 @@
+
+Mail-enabled security groups are used in the same way as security groups are, but with the added feature of a shared mailbox for the groups. Mail-enabled security groups can't be created through the API, but other group operations will still work here. Learn more in the [Manage mail-enabled security groups Exchange article](https://technet.microsoft.com/en-us/library/bb123521(v=exchg.160).aspx).
+
+### Example of security group
+
+```http
+{
+    "@odata.type": "#microsoft.graph.group",
+    "id": "f87faa71-57a8-4c14-91f0-517f54645106",
+    "deletedDateTime": null,
+    "classification": null,
+    "createdDateTime": "2016-07-20T09:21:23Z",
+    "description": "This group is a Security Group",
+    "displayName": "SecurityGroup101",
+    "groupTypes": [],
+    "mail": null,
+    "mailEnabled": false,
+    "mailNickname": "mysg",
+    "preferredLanguage": null,
+    "proxyAddresses": [],
+    "securityEnabled": true
+}
+```
+## Dynamic membership 
+All types of groups can have dynamic membership rules which automatically add or remove members from the group based on user properties. For example, a "Marketing employees" group would include every user with the department property set to "Marketing", so that new marketing employees are automatically added to the group and employees who leave the department are automatically removed from the group. This rule can be specified in a "membershipRule" field during group creation as ```"membershipRule": 'user.department -eq "Marketing"'``` GroupType must also include ```"DynamicMembership"```. The following request creates a new Office 365 group for the marketing employees: 
+
+```http
+POST https://graph.microsoft.com/v1.0/groups
+{
+    "description": "Marketing department folks",
+    "displayName": "Marketing department",
+    "groupTypes": [
+        "Unified",
+        "DynamicMembership"
+    ],
+    "mailEnabled": true,
+    "mailNickname": "marketing",
+    "securityEnabled": false,
+    "membershipRule": 'user.department -eq "Marketing"',
+    "membershipRuleProcessingState": "on"
+}
+```
+
+Learn more about formulating membershipRules in [advanced rules](https://docs.microsoft.com/en-us/azure/active-directory/active-directory-groups-dynamic-membership-azure-portal).
+
+> **Note**: Dynamic membership rules requires the tenant to have a license at tier [Azure Active Directory Premium P1](https://azure.microsoft.com/en-us/pricing/details/active-directory/) or greater.
+
+## Other types of groups
+
+Office 365 groups in Yammer are used to facilitate user collaboration through Yammer posts. This type of group can be returned through a read request, but their posts can't be accessed through the API. To learn more, see [Yammer developer API docs](https://developer.yammer.com/docs).
+
+## Common use cases
```

Comment:
I wonder if there are a few more we should add here, like get events, get conversations, get files and get O365 Groups I'm a member of.